subiquity/cloudinit.py

```diff
@@ -1,12 +1,20 @@
 """Shared cloudinit utility functions"""
 
 import json
+import logging
+
+log = logging.getLogger('subiquity.cloudinit')
 
 
 def get_host_combined_cloud_config() -> dict:
     """Return the host system /run/cloud-init/combined-cloud-config.json"""
     try:
         with open("/run/cloud-init/combined-cloud-config.json") as fp:
-            return json.load(fp)
+            config = json.load(fp)
+            log.debug(
+                "Loaded cloud config from "
+                "/run/cloud-init/combined-cloud-config.json"
+            )
+            return config
```

Comment:
Is it worth logging something in the "we did not use combined cloud config" case?